<ui version="4.0" >
 <class>KeyframeAnimationWidget</class>
 <widget class="QWidget" name="KeyframeAnimationWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Available Keyframe Animations:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_animations" >
     <property name="dragDropMode" >
      <enum>QAbstractItemView::DropOnly</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>FPS</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="m_addAnim" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>81</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_playButton" >
       <property name="text" >
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_removeAnim" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="m_actionAddAnim" >
   <property name="text" >
    <string>Add</string>
   </property>
  </action>
  <action name="m_actionRemoveAnim" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </action>
  <action name="m_actionPlayAnim" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Play</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
